### PR TITLE
docs: Getting started documentation

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
Provides a high-level understanding of Oryx. 
As soon as more documentation is added, we can link from here. 

closes [HRZ-2144](https://spryker.atlassian.net/browse/HRZ-2144)

[HRZ-2144]: https://spryker.atlassian.net/browse/HRZ-2144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: _drafts/oryx/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Oryx is a framework for building composable front-ends, providing a rich library of components, and a design system. The components integrate with Spryker APIs by default, providing a seamless experience for developers.
+
+There are different ways to learn Oryx. If you like to get your hands dirty straight away, you can continue to the [setup guide](./set-up-oryx.md). If you like to learn the concepts, you can continue reading this document, and learn the high-level characteristics and features. More details will be provided in the linked documents.
+
+## Key Characteristics
+
+- **Composable & Configurable** – Oryx components can be composed and configured to fit your project needs.
+- **Extensible & Upgradable** – All components are customizable without losing the ability to upgrade to newer versions.
+- **Platform Agnostic** – Oryx can be integrated into any web technology, it is based on web components and uses plain Typescript code wherever possible.
+- **Component Driven** – Oryx is component centric and uses are based on web components
```
Comment (andriitserkovnyi) [inline _drafts/oryx/getting-started.md:19]: What is `uses` in this component?

Comment (tobi-or-not-tobi) [reply _drafts/oryx/getting-started.md]: badly written, I've rephrased.

**Review by jmartinspryker — CHANGES_REQUESTED** [review]
Submitted changes to oryx overview statement

**Review by jmartinspryker — CHANGES_REQUESTED** [review]

File: _drafts/oryx/getting-started-with-oryx.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Getting started with Oryx
+description: Get a general idea of Oryx
+template: concept-topic-template
+last_updated: Apr 4, 2023
+---
+
+Oryx is a framework for building composable frontends, providing a rich library of components and a design system. The components integrate with Spryker APIs by default, providing a seamless experience for developers.
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply _drafts/oryx/getting-started-with-oryx.md:8]: How about this?

```suggestion
Oryx is a framework that empowers developers to build composable frontends with ease. It provides a rich library of components and a design system, which let developers create modern and visually appealing user interfaces. The components integrate with Spryker APIs by default, providing a seamless experience for developers.
```

[the remark this answers is not in the document]
Comment (tobi-or-not-tobi) [reply _drafts/oryx/getting-started-with-oryx.md:8]: what about:
> a rich library of components including a design system,

[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply _drafts/oryx/getting-started-with-oryx.md:8]: Isn't it the other way around?

[the remark this answers is not in the document]
Comment (tobi-or-not-tobi) [reply _drafts/oryx/getting-started-with-oryx.md:8]: no, there's a lot of components, and parts of it are design system components. 

For example, we have "domain" components (eg "add to cart") that are using design system components (eg a button).

**Review by jmartinspryker — APPROVED** [review]

File: _drafts/oryx/getting-started-with-oryx.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: Getting started with Oryx
+description: Get a general idea of Oryx
+template: concept-topic-template
+last_updated: Apr 4, 2023
+---
+
+Oryx is a framework that empowers developers to build composable frontends with ease. It provides a rich library of components including a design system, which let developers create modern and visually appealing user interfaces. The components integrate with Spryker APIs by default, providing a seamless experience for developers.
```
Comment (VadymSachenko) [inline _drafts/oryx/getting-started-with-oryx.md:8]:
```suggestion
Oryx is a framework that empowers developers to build composable frontends with ease. It provides a rich library of components, including a design system, which lets developers create modern and visually appealing user interfaces. The components integrate with Spryker APIs by default, providing a seamless experience for developers.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**